### PR TITLE
Learning Equality Website changes

### PR DESCRIPTION
**Pull request by khangmach**
Made changes to CKEditor per Richard's instruction.

Per Carine's request:
- Changed hyperlink on the "about page"
- Changed fact and figures/numbers
- Changed footer button styles (donate/contact)
- Added intro text on the Kolibri Landing page, specifically below the Kolibri Intro
- Commented out the donate section on LE main page (under "our impact")

### CODE REVIEW

File: fle_site/apps/about/templates/about/mission.html [hunk 1]
```diff
@@ -26,22 +26,22 @@
...
 
-      <p align="center"><a href="/r/indiegogo/">Help support our crowdfunding campaign so that we can make Kolibri a reality!</a></p>
+      <!--<p align="center"><a href="/r/indiegogo/">Help support our crowdfunding campaign so that we can make Kolibri a reality!</a></p>-->
```
Comment (christianmemije) [inline fle_site/apps/about/templates/about/mission.html]: Remove comment.

File: fle_site/apps/main/templates/main/kickstarter.html [hunk 2]
```diff
@@ -51,12 +51,19 @@
...
                                 <br><br><br><br>
+                                 
                                 <div class="col-md-6">
                                     <div class="intro-heading">Kolibri<span class="hidden-xs hidden-sm" style="font-weight: 100; top:-8px; position: relative;">|</span> </div>
                                 </div>
                                 <div class="col-md-6">
                                      <div class="intro-heading2">The offline app for<br>universal education</div>
                                 </div>
+                                
+                                <br/><br/>
```
Comment (christianmemije) [inline fle_site/apps/main/templates/main/kickstarter.html]: `<br>` not `<br/>`

File: fle_site/apps/main/templates/main/kickstarter.html [hunk 3]
```diff
@@ -51,12 +51,19 @@
...
+                                <div class="col-sm-12">
+                                     <h4 style="text-align:center; font-weight:500;">Watch us take Flight this Summer!</h4>
```
Comment (christianmemije) [inline fle_site/apps/main/templates/main/kickstarter.html]: Add a class instead of inlining style.

**Comment by jtamiace** [comment]
Was the intended change to the "About" redirect supposed to be to this page? Or back to the home page? Currently the about links are taking me to this screenshot

screen shot 2017-03-20 at 10 37 10 am

**Comment by jtamiace** [comment]
screen shot 2017-03-20 at 10 43 10 am

'Jobs' and 'Internships' page have inconsistent data showing (updated parts say 4.3 billion)

**Comment by khangmach** [comment]
@jtamiace think I'm done with changes for now. Will create a quick banner tomorrow as another pull request

**Pull request merged**